editors/vscode/src/wasm_preview.ts

```diff
@@ -192,8 +221,12 @@ function initPreviewPanel(
         context.extensionUri,
         "out/slint_lsp_wasm.js",
     );
+    const default_style = vscode.workspace
+        .getConfiguration("slint")
+        .get("preview.style", "");
     panel.webview.html = getPreviewHtml(
         panel.webview.asWebviewUri(lsp_wasm_url),
+        default_style,
```

Comment:
i'd think this information would be passed from the LSP to the preview using this channel we have now.  But Ok. whatever works.